### ISSUE TITLE
Separation of Shift, CTRL, ALT (Left and Right)

### ISSUE THREAD

**Issue by BlenderBR** [armory3d/armory#1241]
Hello, I don't know if I can make myself understandable ... but for version 0.7, would it  be possible to separate the Shift, Alt and CTRL keys (left and right) as in BGE ??

**Comment by Bugsbane** [comment]
BY this, I think @BlenderBR  means that the Keyboard nodes (or soon, node) could detect either left Ctrl/ right ctrl and left shit / right shift, instead of just "a shift key" or "a ctrl key".

**Comment by BlenderBR** [comment]
Yeah!! That's exactly what I meant... as it happens in the BGE...
